### PR TITLE
choos

### PR DESCRIPTION
**Pull request by itsdoddsy**
i don't even like pikachu

### CODE REVIEW

**Comment by itsdoddsy** [comment]
Me too

File: steely/plugins/choose.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+ERRORS = ['fucking moron',
+          'how about no',
+          'pikachu',]
+
+def main(bot, author_id, message, thread_id, thread_type, *args, **kwargs):
+    try:
+        args = re.findall(r"\w+\b(?<!\bchoose)", message)
+        bot.sendMessage(random.choice(args), thread_id=thread_id, thread_type=thread_type)
+    except IndexError:
```
Comment (devoxel) [inline steely/plugins/choose.py]: Wait so this exception is just for covering your own errors? Can you maybe comment what kind of errors produce these issues?

File: steely/plugins/choose.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import re
+import random
+
+ERRORS = ['fucking moron',
+          'how about no',
+          'pikachu',]
+
+def main(bot, author_id, message, thread_id, thread_type, *args, **kwargs):
+    try:
+        args = re.findall(r"\w+\b(?<!\bchoose)", message)
```
Comment (devoxel) [inline steely/plugins/choose.py]: This regex could be compiled

File: steely/plugins/choose.py (hunk 1 above)
Comment (itsdoddsy) [reply steely/plugins/choose.py]: When someone uses `.choose` but doesn't give anything to choose from it raises IndexError

File: steely/plugins/choose.py (hunk 2 above)
Comment (itsdoddsy) [reply steely/plugins/choose.py]: Fuck off

Comment (itsdoddsy) [reply steely/plugins/choose.py]: you also forget i have next to no knowledge of python

**Comment by byxor** [comment]
Im happy to merge this

File: steely/plugins/choose.py (hunk 2 above)
Comment (iandioch) [reply steely/plugins/choose.py]: I don't think that's a big deal

Comment (devoxel) [reply steely/plugins/choose.py]: not a big deal, but a good olde nit pick

**Pull request merged**